lta/rest_server.py

```diff
@@ -41,26 +41,19 @@
 ASCENDING = pymongo.ASCENDING
 BUNDLE_STATES = ['accessible', 'deletable', 'inaccessible', 'none', 'transferring']
 REMOVE_ID = {"_id": False}
-STR2BOOL_FALSE_SET = {'0', 'f', 'false', 'n', 'no'}
-STR2BOOL_TRUE_SET = {'1', 't', 'true', 'y', 'yes'}
+TRUE_SET = {'1', 't', 'true', 'y', 'yes'}
 
-def now() -> str:
-    """Return string timestamp for current time, to the second."""
-    return datetime.utcnow().isoformat(timespec='seconds')
-
-# https://github.com/symonsoft/str2bool/blob/master/str2bool/__init__.py
-def str2bool(value: str, raise_exc: bool = False) -> Optional[bool]:
-    """Convert a string into a False, None, or True value."""
+def boolify(value: str) -> bool:
+    """Convert a string into a True or False value."""
     if isinstance(value, str):
         value = value.lower()
-        if value in STR2BOOL_TRUE_SET:
+        if value in TRUE_SET:
             return True
-        if value in STR2BOOL_FALSE_SET:
-            return False
+    return False
```

Comment:
A shorter implementation of `boolify` might be:

```python
return isinstance(value, str) and value.lower() in TRUE_SET
```

Reply:
Used this implementation verbatim.